### DISCUSSION TITLE
any way to exclude severity info in workflows ?

### DISCUSSION THREAD

**Discussion by blablareportblabla** [projectdiscovery/nuclei#1894] [Q&A] [answered]
Hi all

Does `es` option work in workflows ? 

e.x.: 

```
id: wordpress-workflow
info:
  name: Wordpress Security Checks
  author: kiblyn11,zomsop82
  description: A simple workflow that runs all wordpress related nuclei templates on a given target.

workflows:

  - template: technologies/wordpress-detect.yaml
    subtemplates:
      - template: vulnerabilities/wordpress/wordpress-rdf-user-enum.yaml
   
```

`vulnerabilities/wordpress/wordpress-rdf-user-enum.yaml` is info template

```
nuclei -w template.yaml  -target https://corporatedealsgov.marriott.com -es info -no-interactsh

produces
[2022-04-26 19:03:51] [wordpress-rdf-user-enum] [http] [info] https://corporatedealsgov.marriott.com/feed/rdf []
```

So. Can we exclude templates in workflows ?

Thanks a lot.

**Comment by ehsandeep** [comment, marked as answer]
@blablareportblabla for the time being, the `severity/s` flag works with workflows, so you can do something like `nuclei -w template.yaml  -target https://corporatedealsgov.marriott.com -s critical,high,medium,low -ni` for the same, we will create additional GH issue to support other filters with workflows.
